testdata/sqllogictests_sqlite/data/setup-test-sqlite-db.sql

```diff
@@ -110,5 +110,35 @@ CREATE TABLE IF NOT EXISTS bikeshare_trips (
     duration_minutes   INT
 );
 
+DROP TABLE IF EXISTS orders;
+CREATE TABLE orders (
+    order_id TEXT,
+    customer_id TEXT,
+    employee_id INT,
+    order_date TIMESTAMP,
+    required_date DATE,
+    shipped_date TIME,
+    ship_via INT,
+    freight REAL,
+    ship_name TEXT,
+    ship_address TEXT,
+    ship_city TEXT,
+    ship_region TEXT,
+    ship_postal_code TEXT,
+    ship_country TEXT
+);
+
 .mode csv
 .import --skip 1 testdata/sqllogictests_datasources_common/data/gcs-artifacts/bikeshare_trips.csv bikeshare_trips
+.import --skip 1 testdata/sqllogictests_datasources_common/data/orders.csv orders
+
+CREATE TABLE IF NOT EXISTS date_test(
+    datetime_a DATETIME,
+    datetime_b DATETIME,
+    text_a TEXT
+);
+
+INSERT INTO date_test (datetime_a, datetime_b, text_a) VALUES
+('2016-07-04', '13:46:17', '2012-12-26 04:58:22'),
+('2022-02-09','23:37:27', '2016-07-11' ),
```

Comment:
I'm not sure what the desired semantics for "timestamp [datetime] without date part" is. 

This is presently an error, and that seems correct.